app/views/keywords/_url_table.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if not urls.empty? %>
```

Comment:
```suggestion
<% if urls.present?? %>
```

Reply:
Nice catch!
Fixed in [`5f004ca` (#47)](https://github.com/suho/google-search-ruby/pull/47/commits/5f004cae025caf7b86ce910239bd5478850fb86a)